app/script/user/UserHandleGenerator.js

```diff
@@ -0,0 +1,115 @@
+  const RANDOM_WORDS_2 = ['alligator', 'alpaca', 'ant', 'antelope', 'asp', 'badger', 'bat', 'bear', 'bee', 'beetle', 'bird', 'bison', 'bobcat', 'buffalo', 'buzzard', 'camel', 'caribou', 'carp', 'cat', 'catfish', 'cheetah', 'clam', 'cobra', 'cod', 'condor', 'cow', 'coyote', 'crane', 'crayfish', 'cricket', 'crow', 'deer', 'dog', 'dolphin', 'donkey', 'dove', 'duck', 'eagle', 'eel', 'elk', 'falcon', 'ferret', 'finch', 'fly', 'fox', 'frog', 'gazelle', 'giraffe', 'gnu', 'goat', 'goose', 'gopher', 'grouse', 'gull', 'halibut', 'hamster', 'hare', 'hawk', 'heron', 'herring', 'horse', 'husky', 'impala', 'jackal', 'jaguar', 'kangaroo', 'koala', 'lemur', 'lion', 'lizard', 'llama', 'lobster', 'mackerel', 'mole', 'moose', 'moth', 'mouse', 'mule', 'mussel', 'newt', 'octopus', 'orca', 'ostrich', 'otter', 'owl', 'ox', 'oyster', 'panda', 'panther', 'parrot', 'pelican', 'penguin', 'pigeon', 'pike', 'pony', 'quail', 'rabbit', 'racoon', 'ram', 'raven', 'salmon', 'sardine', 'seal', 'shark', 'sheep', 'sloth', 'snail', 'snake', 'squid', 'sturgeon', 'swan', 'tiger', 'tilapia', 'toad', 'trout', 'tuna', 'turkey', 'turtle', 'walrus', 'wapiti', 'wasp', 'weasel', 'whale', 'wolf', 'wombat', 'yak', 'zebra'];
+
+  function get_random_word_combination() {
+    return `${z.util.ArrayUtil.random_element(RANDOM_WORDS_1)}${z.util.ArrayUtil.random_element(RANDOM_WORDS_2)}`;
+  }
+
+  /**
+   * Creates variations of the given handle by appending random digits.
+   * @param {string} handle - Input
+   * @param {number} number_of_variations - Number of variants that should be generated
+   * @returns {Array<string>} Handle variations
+   */
+  function generate_handle_variations(handle, number_of_variations = 5) {
+    return new Array(number_of_variations).map(function(value, i) {
+      i += 1;
+      return append_random_digits(handle.slice(0, MAX_HANDLE_LENGTH - i), i);
+    });
+  }
+
+  /**
+   * Appends random digits from 1 to 9 to the end of the string.
+   * @param {string} handle - Input
+   * @param {number} additional_numbers - number of digits to append
+   * @returns {string} String appended with random digits.
+   */
+  function append_random_digits(handle, additional_numbers) {
+    const random_digits = [];
+
+    for (let i = 0; i < additional_numbers; i++) {
+      random_digits.push(z.util.get_random_int(1, 9));
+    }
```

Comment:
Please read the paragraph "Extra code is not the enemy" from: https://dzone.com/articles/using-java-8-please-avoid-functional-vomit